client/client-lib/src/lib.rs

```diff
@@ -552,7 +561,7 @@ impl Client<UserClientConfig> {
             .await?;
 
         // Await acceptance by the federation
-        let timeout = std::time::Duration::from_secs(10);
+        let timeout = std::time::Duration::from_secs(15);
```

Comment:
Is there an issue for that yet?

Reply:
No, this code makes epochs take slightly longer, enough to trip this time-out.

Reply:
I know it's an pre-existing issued, created #368.